<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>739</width>
    <height>558</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SysAdminHelper</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(70, 70, 70, 255));</string>
  </property>
  <widget class="QPushButton" name="pb_ipconfig">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20;
color: rgb(190, 190, 190);
background-color: rgb(57, 57, 57);
border: 2px solid;
border-color: rgb(190, 190, 190);</string>
   </property>
   <property name="text">
    <string>ipconfig</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_tracer">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20;
color: rgb(190, 190, 190);
background-color: rgb(57, 57, 57);
border: 2px solid;
border-color: rgb(190, 190, 190);</string>
   </property>
   <property name="text">
    <string>tracert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_getmac">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20;
color: rgb(190, 190, 190);
background-color: rgb(57, 57, 57);
border: 2px solid;
border-color: rgb(190, 190, 190);
</string>
   </property>
   <property name="text">
    <string>getmac</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_ping">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20;
color: rgb(190, 190, 190);
background-color: rgb(57, 57, 57);
border: 2px solid;
border-color: rgb(190, 190, 190);</string>
   </property>
   <property name="text">
    <string>ping</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_nslookup">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20;
color: rgb(190, 190, 190);
background-color: rgb(57, 57, 57);
border: 2px solid;
border-color: rgb(190, 190, 190);</string>
   </property>
   <property name="text">
    <string>nslookup</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_netstat">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20;
color: rgb(190, 190, 190);
background-color: rgb(57, 57, 57);
border: 2px solid;
border-color: rgb(190, 190, 190);</string>
   </property>
   <property name="text">
    <string>netstat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>560</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>23</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(190, 190, 190);
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>Добро пожаловать в SystemAdminHelper!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(190, 190, 190);
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>Выберите команду</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_author">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>442</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20;
color: rgb(190, 190, 190);
background-color: rgb(57, 57, 57);
border: 2px solid;
border-color: rgb(190, 190, 190);</string>
   </property>
   <property name="text">
    <string>Об авторе</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
